docs/src/main/tut/validated.md

```diff
@@ -296,16 +302,21 @@ val houseNumber = config.parse[Int]("house_number").andThen{ n =>
 ### `withXor`
 The `withXor` method allows you to temporarily turn a `Validated` instance into an `Xor` instance and apply it to a function.
 
-```tut
+```tut:silent
 import cats.data.Xor
 
 def positive(field: String, i: Int): ConfigError Xor Int = {
   if (i >= 0) Xor.right(i)
   else Xor.left(ParseError(field))
 }
+```
 
+Thus.
+
+```tut
 val houseNumber = config.parse[Int]("house_number").withXor{ xor: ConfigError Xor Int =>
   xor.flatMap{ i =>
-    positive(i)
+    positive("house_number", i)
   }
+}
```

Comment:
This compile error wasn't detected because the file ended with a code block in the didn't have its closing `}` so the REPL was left hanging. Nice work isolating a corner case!